src/aat/resources/features/common/case/befta-jurisdiction2-case-creation/Befta_Jurisdiction2_Default_Full_Case_Creation_Data.td.json

```diff
@@ -0,0 +1,190 @@
+          "document_filename": "Elastic Search test Case.png"
+        },
+        "DocumentField1": {
+          "document_url": "http://dm-store:8080/documents/e636eef3-839a-48cd-96f0-dad43e66f5c4",
+          "document_binary_url": "http://dm-store:8080/documents/e636eef3-839a-48cd-96f0-dad43e66f5c4/binary",
+          "document_filename": "Screenshot 2019-11-13 at 3.24.26 pm.png"
+        },
+        "ComplexField": {
+          "ComplexTextField": null,
+          "ComplexFixedListField": null,
+          "ComplexNestedField": {
+            "NestedNumberField": null,
+            "NestedCollectionTextField": []
+          }
+        },
+        "FixedRadioListField": "RADIOVALUE2",
+        "DateTimeField": null,
+        "PhoneUKField": null,
+        "DocumentField7": {
+          "document_url": "http://dm-store:8080/documents/5c14f8ef-887b-41e0-ad01-23959c082311",
+          "document_binary_url": "http://dm-store:8080/documents/5c14f8ef-887b-41e0-ad01-23959c082311/binary",
+          "document_filename": "Screenshot 2019-12-24 at 3.41.04 pm.png"
+        },
+        "NumberField": "111111222222",
+        "MultiSelectListField": [
+          "OPTION5",
+          "OPTION4",
+          "OPTION3"
+        ],
+        "YesOrNoField": "Yes",
+        "EmailField": "testemail@gmail.com",
+        "TextField": "sometextfield",
+        "DateField": "2222-01-02",
+        "TextAreaField": "textarea",
+        "CollectionField": []
+      },
+      "data_classification": {
+        "DocumentField4": "PUBLIC",
+        "MoneyGBPField": "PUBLIC",
+        "FixedListField": "PUBLIC",
+        "DocumentField3": "PUBLIC",
+        "AddressUKField": {
+          "classification": "PUBLIC",
+          "value": {
+            "AddressLine1": "PUBLIC",
+            "AddressLine2": "PUBLIC",
+            "AddressLine3": "PUBLIC",
+            "PostTown": "PUBLIC",
+            "County": "PUBLIC",
+            "PostCode": "PUBLIC",
+            "Country": "PUBLIC"
+          }
+        },
+        "DocumentField6": "PUBLIC",
+        "DocumentField5": "PUBLIC",
+        "DocumentField2": "PUBLIC",
+        "DocumentField1": "PUBLIC",
+        "FixedRadioListField": "PUBLIC",
+        "DateTimeField": "PUBLIC",
+        "PhoneUKField": "PUBLIC",
+        "DocumentField7": "PUBLIC",
+        "NumberField": "PUBLIC",
+        "MultiSelectListField": "PUBLIC",
+        "YesOrNoField": "PUBLIC",
+        "EmailField": "PUBLIC",
+        "TextField": "PUBLIC",
+        "DateField": "PUBLIC",
+        "TextAreaField": "PUBLIC",
+        "CollectionField": {
+          "classification": "PUBLIC",
+          "value": []
+        }
+      },
+      "after_submit_callback_response": null,
+      "callback_response_status_code": null,
+      "callback_response_status": null,
+      "delete_draft_response_status_code": null,
+      "delete_draft_response_status": null,
+      "security_classifications": {
+        "DocumentField4": "PUBLIC",
```

Comment:
This can also be extended from existing file.

Reply:
Need Clarity.

Reply:
Done.